src/sections/5.speakers.js

```diff
@@ -145,10 +159,16 @@ export default () => {
                 target="speaker-tan-li-yuan-marcus-siddhant"
               />
               <SpeakerImage
-                name="Siddhant Shrivastava"
-                position="Emerging Tech Researcher at iTrust Centre for Research in Cyber Security, SUTD"
-                fixed={data.siddhant.childImageSharp.fixed}
-                target="speaker-tan-li-yuan-marcus-siddhant"
```

Comment:
Oh no, this should be put back

Reply:
putting it back. nice catch

Reply:
Wah good catch. Sleepy liao cannot see anything wahaha